### ISSUE TITLE
echo-sigar-libs and echo-trace-play-2.3.9_2.11 not found

### ISSUE THREAD

**Issue by jonathanglima** [typesafehub/activator#1028]
My activator's giving me this error below. I really couldn't find echo-trace-play-2.3.9_2.11 and echo-sigar-libs on version 0.1.15 on bintray repos, or anywhere. Any guesses?

Read from stdout: (*:update) sbt.ResolveException: unresolved dependency: com.typesafe.trace#echo-trace-play-2.3.9_2.11;0.1.15: not found
Read from stdout: unresolved dependency: com.typesafe.trace#echo-sigar-libs;0.1.15: not found

**Comment by jonathanglima** [comment]
Solved.

I had to do a git checkout the project/play-fork-run.sbt file on git so it could return the line below to 2.3.8:

> addSbtPlugin("com.typesafe.play" % "sbt-fork-run-plugin" % "2.3.9")

After that I reseted the "activator ui" command. It ran the whole upgrade to 2.3.9 again and now it worked.

Probably some instability on the repos...?

**Comment by corruptmemory** [comment]
Thanks for the update.  I'm a little confused.  You said: 'I had to do a git checkout the "project/play-fork-run.sbt"' Is this all against your own project or a template?

**Comment by jonathanglima** [comment]
My own template. Sorry, I guess I wasn't clear enough.

**Comment by corruptmemory** [comment]
ok, so your current setting is:

> addSbtPlugin("com.typesafe.play" % "sbt-fork-run-plugin" % "2.3.9")

Yes?  Or is it:

> addSbtPlugin("com.typesafe.play" % "sbt-fork-run-plugin" % "2.3.8")

Just trying to figure out what artifact is missing from bintray.  Thanks.

**Comment by jonathanglima** [comment]
My setting was 2.3.8 but activator ui changed it to 2.3.9. That's when the error happened.

I had to change it back to 2.3.8 so the update process could run all over again. After the second update finished the error stopped and then my setting changed again to 2.3.9 probably because activator ui edited it.

**Comment by corruptmemory** [comment]
Thanks for the prompt reply.  I'll dig into this.

**Comment by w3apps** [comment]
This should be an opened issue.
Read from stdout: (*:update) sbt.ResolveException: unresolved dependency: com.typesafe.trace#echo-sigar-libs;0.1.15: not found

The version change thing didn't work for me. Any other ideas?

**Comment by w3apps** [comment]
A followup to my previous comment, I managed to fix this by changing changing the sbt-echo version in "project/inspect.sbt":
addSbtPlugin("com.typesafe.sbt" % "sbt-echo" % "0.1.15")
Changed the above to:
addSbtPlugin("com.typesafe.sbt" % "sbt-echo" % "0.1.14")

I also noticed there's no 0.1.15 version here:
http://dl.bintray.com/typesafe/maven-releases/com/typesafe/trace/echo-sigar-libs/

**Comment by jonathanglima** [comment]
Nice, @w3apps.

I tried that fix in the first place and for me it didn't worked.

Should I reopen it?

**Comment by w3apps** [comment]
Yes please, the error shouldn't be there in the first place, we're just trying hacky stuff to fix it.

**Comment by rohit5ram** [comment]
None of the above fixes worked for me. Can some one help ? 
I followed work around mentioned by @jonathanglima  and @w3apps . But none worked for me. I got a new resolve exception sbt.ResolveException: unresolved dependency: com.typesafe.trace#trace-play-2.3.0_2.10;0.1.14: not found. And i dont find the version in place http://dl.bintray.com/typesafe/maven-releases/com/typesafe/trace/trace-play-2.3.0_2.10/ 
Where can i edit this dependency ?

**Comment by alex88** [comment]
The problem is that echo-sigar-libs 0.1.14 doesn't let you install an akka version supported by activator's ui monitoring panel so you can't monitor actors

**Comment by rohit5ram** [comment]
Oh Thank you @alex88 .  That's why every error resolved when i shifted back to simply using activator run

**Comment by alex88** [comment]
I don't get why that version is required but it's not there, someone has typesafe subscription to check if it's there? (and so the monitor thing is just for paid customers)
